### ISSUE TITLE
Horizontal Gates

### ISSUE THREAD

**Issue by mRichard629** [PC-Logix/LanteaCraft#319]
having the ability to have a horizontal gate would be kind of cool

**Comment by Timmy94W** [comment]
Yeah, that will be awesome. Dial to Horizontal gate and the normal ones

**Comment by mRichard629** [comment]
It's conical as well, they had one in the fifth episode of the first season

**Comment by Zunth5** [comment]
> It's conical as well, they had one in the fifth episode of the first season 

Well thats true...    but they also blew up a sun in the series, but you don't see Lanteacraft adding sun-bound Stargate catapults.

I don't disagree that it would be awesome to have horizontally constructed gates, but in the context of minecraft I don't think this would be very useful unless it was used to hide buried stargates in the world under cover stones or something.

**Comment by mRichard629** [comment]
I wasn't being snobbish adding the conical bit. I was watching the episode when I put the idea up. You also don't need to be a jerk about it.

**Comment by Timmy94W** [comment]
To be honest: i like the idea. In the simplest way you can use it to flood
you base when you place at the ground oft the ocean or huge lava lakes and
so you have a use for the Iris, whitch takes damage, when Lava tries to
flood from gate to gate
Am 20.05.2015 22:34 schrieb "mRichard629" <notifications@github.com>:

**Comment by AfterLifeLochie** [comment]
> having the ability to have a horizontal gate would be kind of cool

Lying down gates will be impossibly difficult to use. If you fall back through an event horizon which you've come out of and `oneWayTravel` is on, you *will* die.

> but in the context of minecraft I don't think this would be very useful unless it was used to hide buried stargates in the world under cover stones or something.

I would tend to agree with this, simply on the basis of the requirement to put the gates down a certain way to prevent travelling objects from falling back in - it means exit gates and entry gates are essentially omni-directional. It's also worth noting that obstructing Stargates is no longer a good idea as the transient will just obliterate obstructing blocks (or items, or entities) now.

> you can use it to flood you base when you place at the ground oft the ocean or huge lava lakes and so you have a use for the Iris, whitch takes damage, when Lava tries to flood from gate to gate

No, because fluids don't (and won't) pass through the Stargate for a variety of reasons I've already discussed before. Unless Minecraft's fluid system becomes sane (which it won't), I physically can't implement something like this without "*Weird Side Effects*".

>I wasn't being snobbish adding the conical bit. I was watching the episode when I put the idea up. You also don't need to be a jerk about it.

If you get aggressive, I will just lock this. People are certainly welcome to express an opinion so that we can have fair discussion about suggestions - in particular ones I'm not particularly sold on. If you're going to be a jerk yourself, I will send you away.

**Comment by Zunth5** [comment]
I will apologise for the connical bit, as it was meant to be more humoras than snobbish. I did not intend it as a jab.

**Comment by Zunth5** [comment]
Did you just imply that the stargate event horizons will now destroy blocks placed in the world?

**Comment by mRichard629** [comment]
I apologize as well, tone, intent and inflection don't come across the Internet well. Your ideas make sense too. My vision was to have a conveyor belt dropping things into the gate as a work around for not being able to have one go through an open gate into a storage room or something like that

**Comment by AfterLifeLochie** [comment]
> Did you just imply that the stargate event horizons will now destroy blocks placed in the world?

Yes. Once irises become a thing, the Stargate will obliterate things which obstruct or otherwise impede it's functional operation.

> My vision was to have a conveyor belt dropping things into the gate as a work around for not being able to have one go through an open gate into a storage room or something like that

I honestly don't think Stargates are well suited for this purpose. It seems very corner-case to have a Stargate lying face-up or face-down, which makes me wonder if it's worth implementing. Remember, these are huge dialling machines, not Aperture Science portals here.

**Comment by Cronosus** [comment]
i post something like that a year back i think :D
fluids can't travel throught gate, but stargate inside water/lava (maybe pattern option to make parts of gate like inside water/lava will be good) is not impossible, but now are stargate deformed, when is something inside.
lying stargate will be good too
yes, only for oneway travel, because you die, when you fall back inside gate :D... but why not? :P

**Comment by mRichard629** [comment]
I just thought it was a nifty idea
